lib/preek/ext/smell_warning.rb

```diff
@@ -10,7 +10,7 @@ def smell_method
     end
 
     def smell_string
-      "#{smell_method} #{message} (#{subclass}) at lines #{Array(lines)*','}"
+      "#{smell_method} #{message} (#{smell_detector.class}) at lines #{Array(lines)*','}"
```

Comment:
and this to `"#{smell_method} #{message} (#{smell_type}) at lines #{Array(lines)*','}"`
the specs should pass without changes.